packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -146,6 +146,18 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         _;
     }
 
+    /**
+     * Modifies an EM method so that only a given address can invoke it.
+     */
+    modifier onlyCallableBy(
```

Comment:
Should add tests to ensure this is applied to `ovmomvSETCODE` and `ovmSETSTORAGE`.